lms/djangoapps/grades/tasks.py

```diff
@@ -24,9 +26,26 @@
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
 def recalculate_subsection_grade(
+        # pylint: disable=unused-argument
         user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
 ):
     """
+    Shim to allow us to modify this task's signature without blowing up production on deployment.
+    """
+    _recalculate_subsection_grade.apply(
+        kwargs=dict(
+            user_id=user_id,
+            course_id=course_id,
+            usage_id=usage_id,
+            only_if_higher=only_if_higher,
+            task_queued_time=kwargs.get('task_queued_time', 0),  # If no time, assume 1/1/1970
+            score_deleted=kwargs['score_deleted'],
+        )
+    )
+
+@task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
+def _recalculate_subsection_grade(**kwargs):
```

Comment:
@jibsheet thoughts on this method of collapsing our task signature down to a single `**kwargs`? We're going to be removing `weighted_earned` and `weighted_possible` anyways, so I figure why not go whole-hog.